src/modality-specific-files/electromyography.md

```diff
@@ -0,0 +1,701 @@
+    each modality’s acquisition devices.
+
+-   Storing the acquisition time (relative to a common clock source) of the first data point
+    of each modality’s recording in the acq_time column of the *_scans.tsv file.
+    Note that the BIDS date time format allows optional fractional seconds,
+    which SHOULD be used to maximize the precision of the synchronization.
+
+-   Record the start- and stop-time of one modality’s recording (relative to
+    the other modality’s clock) in the *_events.tsv file for the other modality.
+
+It is up to the user to decide which approach is most appropriate given their experimental setup,
+and to identify which modality will serve as the 'main' clock source for synchronization purposes
+(or whether a separate external clock source is used).
+
+### Sidecar JSON (`*_emg.json`)
+
+For consistency between studies and institutions,
+we encourage users to extract the values of metadata fields from the actual raw data.
+Whenever possible, please avoid using ad hoc wording.
+
+These fields MUST be present:
+
+<!-- This block generates a metadata table.
+These tables are defined in
+  src/schema/rules/sidecars
+The definitions of the fields specified in these tables may be found in
+  src/schema/objects/metadata.yaml
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_sidecar_table("emg.EMGRequired") }}
+
+Note the `RecordingType`, which depends on whether the data stream on disk is interrupted or not.
+Continuous data is by definition 1 segment without interruption.
+Epoched data consists of multiple segments that all have the same length
+(for example, corresponding to trials) and that have gaps in between.
+Discontinuous data consists of multiple segments of different length,
+for example due to a pause in the acquisition.
+
+These fields SHOULD be present:
+
+<!-- This block generates a metadata table.
+These tables are defined in
+  src/schema/rules/sidecars
+The definitions of the fields specified in these tables may be found in
+  src/schema/objects/metadata.yaml
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_sidecar_table("emg.EMGRecommended") }}
+
+These fields MAY be present:
+
+<!-- This block generates a metadata table.
+These tables are defined in
+  src/schema/rules/sidecars
+The definitions of the fields specified in these tables may be found in
+  src/schema/objects/metadata.yaml
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_sidecar_table("emg.EMGOptional") }}
+
+Note that the date and time information SHOULD be stored in the study key file
+([`scans.tsv`](../modality-agnostic-files/data-summary-files.md#scans-file)).
+Date time information MUST be expressed as indicated in [Units](../common-principles.md#units).
```

Comment:
It is not clear which date and time information is being referred to here.

Reply:
Looks like this was copied over from EEG, where it also doesn't seem to refer to anything.

```suggestion
```